disco_aws_automation/disco_metanetwork.py

```diff
@@ -35,9 +34,10 @@ class DiscoMetaNetwork(object):
     def __init__(self, name, vpc):
         self.vpc = vpc
         self.name = name
-        self._route_table = None  # lazily initialized
+        self._centralized_route_table_loaded = False
+        self._centralized_route_table = None  # lazily initialized
         self._security_group = None  # lazily initialized
-        self._subnets = None  # lazily initialized
+        self._disco_subnets = None  # lazily initialized
```

Comment:
I created a DiscoSunet class to encapsulate the logic dealing with all the things that are associated with a subnet, which include its own route table, NAT gateway, cidr.